docker/mu-plugins/conflicting-plugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Plugin Name: Conflicting Plugin
+ * License: GPL2+
+ */
+error_log( 'loading conflicting plugin' );
+/**
+ * THIS CODE WOULD NEED TO BE DUPLICATED IN EACH PLUGIN...
+ */
+if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
+	function jetpack_enqueue_library( $class_name, $version, $path ) {
+		global $jetpack_libraries;
+		if ( ! is_array( $jetpack_libraries ) ) {
+			$jetpack_libraries = array();
+		}
+		if ( ! isset( $jetpack_libraries[ $class_name ] ) ) {
+			$jetpack_libraries[ $class_name ] = array();
+		}
+		$jetpack_libraries[ $class_name ][] = array( 'version' => $version, 'path' => $path );
```

Comment:
Why not do the version comparison here and just overwrite the existing entry if the new entry has a higher version number? That way, you don't have to do any version comparisons at autoload time, since you never plan on loading older versions anyway.